hometask1/tests/test_int_object.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import random
+import sys
+
+
+class TestException(Exception):
```

Comment:
```
hometask1/tests/test_int_object.py:6
  2020-1-Atom-QA-Python-D-Maksimov/hometask1/tests/test_int_object.py:6: 
PytestCollectionWarning: cannot collect test class 'TestException' because it has a __init__ constructor (from: hometask1/tests/test_int_object.py)
    class TestException(Exception):

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

Reply:
Переименовал класс exeption